cmd/kwil-admin/cmds/migration/network_status.go

```diff
@@ -0,0 +1,72 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/kwilteam/kwil-db/cmd/common/display"
+	"github.com/kwilteam/kwil-db/cmd/kwil-admin/cmds/common"
+	"github.com/kwilteam/kwil-db/core/types"
+	"github.com/spf13/cobra"
+)
+
+func networkStatusCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "network-status",
+		Short: "Get the migration status of the network.",
+		Example: `# Get the migration status of the network.
+		kwil-admin migrate network-status`,
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			clt, err := common.GetAdminSvcClient(ctx, cmd)
+			if err != nil {
+				return display.PrintErr(cmd, err)
+			}
+
+			status, err := clt.MigrationStatus(ctx)
```

Comment:
What will happen if we run this command against a new network, or against a network that does not have a migration running?

My concern is: if you run this command against a network in activation or in migration, it will work as expected. However, if you run this command against a node that is in genesis migration or not in migration, then it will show the start and end height of the old (either in-process or already migrated) migration.

IMO, it should:
- function as it currently does for activation and in-migration
- Show no start height, end height, or current height if there is no migration active
- Show no start height, end height, or current height if in genesis migration. This is because the end height here would be relevant to the old network, and not the new network, and thus would be misleading and confusing to consumers

Reply:
```
if m.Status == types.NoActiveMigration {
		return []byte("No active migration on the network."), nil
	}

	if m.Status == types.GenesisMigration {
		return []byte("Genesis migration in progress."), nil
	}```
Yeah we don't show height information if we are in the genesis migration or when there is no migration,

Reply:
Ah I see, my bad I am dense

Reply:
Is it easy to tell whether the network never had a genesis migration to begin with / if it did? The reason being is that if you get `NoActiveMigration`, it would be helpful to tell the user that their previous migration was complete, instead of just saying there isn't one.

If there was never a genesis migration:

```
$ kwil-admin migrate network-status
No active migration on the network.
```

If there was a genesis migration but it finished:

```
$ kwil-admin migrate network-status
Genesis migration complete.
```

If this is hard to do, no worries, but if it is easy it would be nice.

Reply:
ok done